### DISCUSSION TITLE
Expand Selection seems to not work correctly in <template> tags

### DISCUSSION THREAD

**Discussion by ddenev** [vuejs/language-tools#1464] [Q&A]
How can I configure vscode so that "Expand/Shrink Selection" (`Shift+Alt+Right/LeftArrow`) works properly in `.vue` files?

I'm not sure this relates to Volar but "Expand Selection" works properly in `.js` and `.html` files. Here is how it works in `.vue` file:

1. Initial cursor position:
![image](https://user-images.githubusercontent.com/1393766/173776391-95135f0c-3775-49be-b4e2-475f90a1f61c.png)
2. After 1st `Shift+Alt+RightArrow`:
![image](https://user-images.githubusercontent.com/1393766/173776512-7dfa79cd-1503-432f-95ba-b78080f8cd4d.png)
3. After 2nd `Shift+Alt+RightArrow`:
![image](https://user-images.githubusercontent.com/1393766/173776623-653f470d-b8b9-454a-827f-f164c36407ce.png)

I would expect it to select `group.children_ids` on the 2nd expansion, not the whole tag.
This only happens in the `<template>` tag and works OK in the `<script>` and `<style>` tags.

**Comment by johnsoncodehk** [comment]
Hi @ddenev, thanks to report this problem, this is a bug and nice to have an issue to tracking.

**Reply by johnsoncodehk** [reply]
Track #1465.
